RELEASE.md

```diff
@@ -41,58 +41,68 @@ with `make upgrade-npm-deps`
 
 ### 1. Prepare your release
 
-You should start to create a branch that follows the pattern `release/v<X.Y>`. Release candidates and patch releases
+#### Create a release branch
+
+Create a branch that follows the naming pattern `release/v<X.Y>` and includes the changes you intend to release (usually the latest from `main`). Push it to github. You will use this branch as the base in the next step.
+
+Release candidates and patch releases
 for any given major or minor release happen in the same `release/v<major>.<minor>` branch. Do not
 create `release/<version>` for patch or release candidate releases.
 
-- Do this in a proper PR pointing to the release branch as this gives others the opportunity to chime in on the release
-  in general and on the addition to the changelog in particular.
+#### Create a PR with the changes
 
+- Create a branch with the changes for the release (e.g. `yourname/release-v<major>.<minor>.<patch>`). 
 - Update the file `VERSION` with the new version to be created.
-
-- Update the file `CHANGELOG.md` :
-
-```bash
-make generate-changelog
-```
-
-Note: it will generate a first changelog version based on the git history. You should review what has been generated.
-Error can happen ;)
-
-- Entries in the `CHANGELOG.md` are meant to be in this order:
-
-* `[FEATURE]`
-* `[ENHANCEMENT]`
-* `[BUGFIX]`
-* `[BREAKINGCHANGE]`
-
-As we have many libraries we publish, it's better if you also put a clear indication about what library is affected by
+- Generate `CHANGELOG.md` updates based on git history:
+  ```bash
+  make generate-changelog
+  ```
+- Review the generated `CHANGELOG.md` for valid output. Things to check include:
+  - Entries in the `CHANGELOG.md` are meant to be in this order:
+    * `[FEATURE]`
+    * `[ENHANCEMENT]`
+    * `[BUGFIX]`
+    * `[BREAKINGCHANGE]`
+  - Entries that map to a pull request should include a pull request number.
+  - As we have many libraries we publish, it's better if you also put a clear indication about what library is affected by
 these changes.
-
-- Update the different `package.json` with the corresponding version:
-
-```bash
-make bump-version
-```
+  - Consumers understand how to handle breaking changes either through the messaging in the changelog or through the linked pull requests.
+- Update the `package.json` files for all packages with the corresponding version:
+  ```bash
+  make bump-version
+  ```
+- Push the branch to github and create a pull request with the release branch as the base. This gives others the opportunity to chime in on the release,
+  in general, and on the addition to the changelog, in particular. 
+  - It's also helpful to drop a link to the release PR in [#perses-dev](https://matrix.to/#/#perses-dev:matrix.org) to get extra visibility.
```

Comment:
@Nexucis the mdox docs validator is complaining about this link because it's a duplicate. It seemed like maybe they fixed this behavior in https://github.com/bwplotka/mdox/pull/26/files, but I think we're on the latest version, so maybe I'm missing something. Any thoughts? If there isn't an obvious fix, I can remove the link to avoid the issue.

Reply:
so far I don't see an obvious fix excepting putting in the file `.mdox.validate.yaml` the link to ignore it. But that means it won't check anymore if the link is valid 😐

Reply:
For this case, I'm just going to take out the link, but I created https://github.com/perses/perses/issues/1122 for us to look into this a little more deeply when we have time.